### ISSUE TITLE
Frontend Tooling

### ISSUE THREAD

**Issue by mo-martinwilson** [mo-martinwilson/yolo-atlas#2]
- [ ] vite
- [ ] HTMX
- [ ] leaflet-html
- [ ] linting (prettier, eslint)
- [ ] Storybook
- [ ] testing (vitest)

**Comment by andrewgryan** [comment]
Can we skip front end tooling and ship a prototype?

**Comment by mo-martinwilson** [comment]
Good idea, I'll throw together a couple of wireframes and user flow diagrams as PNGs into a design/ directory at the root of the repo for some prototype ideas
